src/bundle/Resources/public/js/scripts/helpers/text.helper.js

```diff
@@ -7,7 +7,21 @@
         return stringTempNode.innerHTML;
     };
 
+    const escapeDataset = (str) => {
+        if (str === null) {
+            return '';
+        }
+
+        return String(str)
+            .replace(/&/g, '&amp;')
+            .replace(/</g, '&lt;')
+            .replace(/>/g, '&gt;')
+            .replace(/"/g, '&quot;')
+            .replace(/'/g, '&#039;');
```

Comment:
Do we need to replace `<` or `>`? 🤔

Reply:
I mapped the htmlspecialchars functions from PHP and more or less what e('html_attr') does in twig